### PR TITLE
py sol for cf-632C

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

AC: https://codeforces.com/contest/632/submission/292353570

### CODE REVIEW

File: solutions/silver/cf-632C.mdx [hunk 1]
```diff
@@ -123,4 +123,28 @@ public class Smallest {
...
+```py
+from functools import cmp_to_key
+
+n = int(input())
+inps = []
+for _ in range(n):
+	inps.append(input())
+
+
+def compare(x, y):
```
Comment (SansPapyrus683) [inline solutions/silver/cf-632C.mdx:137]: type annotations

File: solutions/silver/cf-632C.mdx [hunk 2]
```diff
@@ -123,4 +123,28 @@ public class Smallest {
...
+def compare(x: str, y: str) -> int:
+	if x + y > y + x:  # y should come before x
+		return 1
+	elif x + y < y + x:  # x should come before y
+		return -1
+	return 0  # x and y are equal
+
+
+inps.sort(key=cmp_to_key(compare))
+print("".join(inps))
```
Comment (SansPapyrus683) [inline solutions/silver/cf-632C.mdx:146]:
```suggestion
print("".join(sorted(inps, key=cmp_to_key(compare))))

**Comment by SansPapyrus683** [comment]
send ss of ac again

**Comment by freakin23** [comment]
AC: https://codeforces.com/contest/632/submission/292358712

**Pull request merged**